### ISSUE TITLE
Widget agenda

### ISSUE THREAD

**Issue by alexsegura** [nuitdebout/wordpress#141]
Il faudrait faire un widget agenda qui utilise OpenAgenda. 

Un seul paramètre : le nom de la ville. 
Affiche les évènements du jour sous forme de liste simple.

**Comment by fadomire** [comment]
it is done it seems. reopen if needed.

**Comment by alexsegura** [comment]
Non, ici on parle d'un [widget](https://codex.wordpress.org/Widgets_API), donc utilisable via l'outil "Personnaliser" de Wordpres
